docs/topics/downloader-middleware.rst

```diff
@@ -1074,13 +1074,12 @@ Protego parser
 
 `Protego <https://github.com/scrapy/protego>`_ is a pure-Python robots.txt_ parser.
 The parser is fully compliant with `Google's Robots.txt Specification
-<https://developers.google.com/search/reference/robots_txt>`_ hence supports wildcard 
+<https://developers.google.com/search/reference/robots_txt>`_ hence supports wildcard
 matching, and uses the length based rule similar to `Reppy <https://github.com/seomoz/reppy/>`_.
+Scrapy uses this parser by default.
 
 In order to use this parser:
 
-* Install `Protego <https://github.com/scrapy/protego>`_ by running ``pip install protego``
```

Comment:
currently you still need to install protego manually; this issue is hidden on CI because protego is included in all requirements.txt files. To fix it, protego needs to be added to setup.py install_requires.